astra/src/main/java/com/slack/astra/bulkIngestApi/opensearch/BulkApiRequestParser.java

```diff
@@ -111,6 +111,10 @@ public static Trace.Span fromIngestDocument(
           ByteString.copyFromUtf8(
               String.valueOf(sourceAndMetadata.get(LogMessage.ReservedField.TRACE_ID.fieldName))));
       sourceAndMetadata.remove(LogMessage.ReservedField.TRACE_ID.fieldName);
+    } else if (sourceAndMetadata.get("traceId") != null) {
```

Comment:
astra expects trace id to come in as `trace_id` to put into its trace model properly. the otel spec expects and exports it as `traceId`
https://opentelemetry.io/docs/specs/otel/trace/api/#spancontext

I dont necessarily think this is the right fix to be compatible with OTLP and otel collector, but this is an example that works. and i'd like to open a discussion and get thoughts/ feedback.

this ignores understanding `spanId` because from my digging through the code, span_id is not reserved and displaying self ingested spans (in the zipkin grafana plugin) works with only this change. even though otlp exported spans have `spanId` rather than `span_id`